tests/conftest.py

```diff
@@ -38,3 +40,29 @@ def load(cls) -> dict:
     monkeypatch.setattr('torrt.utils.TorrtConfig', MockConfig)
 
     return settings
+
+
+def pytest_configure(config):
```

Comment:
Let's just install this requirement for test, or stub it entierly, e.g. with pytest-stub

Reply:
Oh, in this case, I guess I need your assistance here.

I made my best in terms of non-destroying original intentions in both tests and plugins. But I have no idea how to mock this bot in a proper way.

Reply:
You may want to try:
1. install pytest-stub and replace the entire module with the stub
2. add bot library to tests_require in setup.py
3. mock module with magicMock manually (since we don't need to test the bot library, itself but only our code)

Reply:
Reverted all my `pytest` trickery. 
used `MagickMock` on everything used in `telegram_bot`. 

So basically, we are back to where we started in terms of testing this part. 
I'll probably add couple of tests to cover newly added behaviour